src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -16,55 +25,101 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
     constructor(props: any) {
         super(props);
         this.state = {
-            isEditingDisabled: false,
+            isEditingDisabled: true,
+            isDirty: false,
```

Comment:
This doens't say to me what this variable means.

isSaveDisabled?

Reply:
In my opinion variable names should not be set to what they affect, the name should be set to what the variable means. This variable tells if the current object opened is dirty. It tells if there have been modifications to the object, and these changes have not been synchronised to the database.

Reply:
Hmm, I see your point why it is "isDirty". Haven't used that term before. How about "hasModifications"?

Reply:
Also generally about the naming: naming with what the variable does/where it affects can also mean at the same time what it means :P